### PR TITLE
implemented the library

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: liba.js [hunk 1]
```diff
@@ -1 +1,102 @@
...
+
+function isNumber(obj) {
+    return (typeof obj === 'number') || (obj instanceof Number);
+}
+
+function isString(obj) {
+    return (typeof obj === 'string') || (obj instanceof String);
+}
+
+function IsFunction(obj) {
```
Comment (siarheikarabko) [inline liba.js:23]: Please, follow the naming convention.

File: liba.js [hunk 2]
```diff
@@ -1 +1,102 @@
...
+    return (obj instanceof Function);
+}
+
+function isUndefined(obj) {
+    return (typeof obj === "undefined");
```
Comment (siarheikarabko) [inline liba.js:28]: Can't we use `obj === undefined` ?

File: liba.js [hunk 3]
```diff
@@ -1 +1,102 @@
...
+}
+
+function isNull(obj) {
+    return (obj === null);
+}
+
+function first(arr) {
+    if (isArray(arr)) {
+        return arr[0];
+    } else {
```
Comment (siarheikarabko) [inline liba.js:38]: You don't need `else` block, the code inside never executes if `arr` is `Array`. Please, review the other methods.

Comment (siarheikarabko) [reply liba.js:38]: Also, please consider a possible case when an input array has no elements.

File: liba.js [hunk 4]
```diff
@@ -1 +1,102 @@
...
+    }
+}
+
+function skip(arr, number) {
+    if (!isArray(arr)) {
+        throw "Argument Exception (arr should be an array)";
+    }
+
+    if (number < 0) {
+        throw "The number can't be less than zero";
```
Comment (siarheikarabko) [inline liba.js:57]: Actually, `slice` allows negative values, please review.

File: liba.js [hunk 5]
```diff
@@ -1 +1,102 @@
...
+        },
+
+        skip: function(func) {
+            this.localArr = this.localArr.filter(function(el) {
+                return !func(el);
+            });
+            return this;
+        },
+
+        skipByIndex: function(func) {
```
Comment (siarheikarabko) [inline liba.js:95]: Why do you need this separate method?

File: liba.js (hunk 1 above)
Comment (kovpaka) [reply liba.js:23]: My bad, fixed.

Comment (kovpaka) [reply liba.js:23]: My bad, fixed.

File: liba.js (hunk 2 above)
Comment (kovpaka) [reply liba.js:28]: Fixed.

File: liba.js (hunk 3 above)
Comment (kovpaka) [reply liba.js:38]: Added the exception.

File: liba.js (hunk 4 above)
Comment (kovpaka) [reply liba.js:57]: Fixed.

File: liba.js (hunk 5 above)
Comment (kovpaka) [reply liba.js:95]: Just for fun) Removed.

**Pull request merged**